### PR TITLE
adding analytics,business_services,rulesets and service_dependencies

### PR DESCRIPTION
**Pull request by stmcallister**

### CODE REVIEW

File: lib/pd.js [hunk 1]
```diff
@@ -133,6 +133,72 @@ function Client(apiToken, tokenType, options) {
...
+        },
+        getIncidentsData: async () => {
+            try {
+                return await request(this.options('/analytics/raw/incidents', 'GET', null, null))
+            }
+            catch (err) {
+                throw new Error(err)
+            }
+        },
+    }
```
Comment (kmartin-isp) [inline lib/pd.js:155]: Can you remove the leading / from the options section `/analytics/raw/incidents`

File: lib/pd.js [hunk 2]
```diff
@@ -809,6 +961,44 @@ function Client(apiToken, tokenType, options) {
...
+    },    
+    getTechnicalServiceDependencies: async(id) => {
+        try {
+            return await request(this.options('/service_dependencies/technical_services/'+ id, 'POST', null, null))
+        }
+        catch (err) {
+            throw new Error(err)
+        }  
+    }
+};
```
Comment (kmartin-isp) [inline lib/pd.js:999]: Can you also remove the leading / from options section here too `/service_dependencies/technical_services/`

**Comment by stmcallister** [comment]
Good catch! Leading slashes have been removed! Also, I didn't get to these, but Response Plays also just released new endpoints.

**Comment by kmart2234** [comment]
I'll get the Response Plays added and merged, then cut a new release.

**Pull request merged**